examples/tutorials/tutorial_2D_30p30n.py

```diff
@@ -0,0 +1,140 @@
+                boundary_layer_first_layer_thickness=4.9536767e-06,
+            ),
+            refinement_factor=1.35,
+            gap_treatment_strength=0.5,
+            volume_zones=[farfield],
+            refinements=[
+                fl.UniformRefinement(name="refinement1", spacing=0.1, entities=[cylinders[0]]),
+                fl.UniformRefinement(name="refinement2", spacing=0.15, entities=[cylinders[1]]),
+                fl.UniformRefinement(name="refinement3", spacing=0.225, entities=[cylinders[2]]),
+                fl.UniformRefinement(name="refinement4", spacing=0.275, entities=[cylinders[3]]),
+                fl.UniformRefinement(name="refinement5", spacing=0.325, entities=[cylinder5]),
+                fl.SurfaceRefinement(name="wing", max_edge_length=0.74, faces=[geometry["wing"]]),
+                fl.SurfaceRefinement(
+                    name="flap-slat",
+                    max_edge_length=0.55,
+                    faces=[geometry["flap"], geometry["slat"]],
+                ),
+                fl.SurfaceRefinement(
+                    name="trailing",
+                    max_edge_length=0.36,
+                    faces=[
+                        geometry["wingTrailing"],
+                        geometry["flapTrailing"],
+                        geometry["slatTrailing"],
+                    ],
+                ),
+                fl.SurfaceEdgeRefinement(
+                    name="edges",
+                    method=fl.HeightBasedRefinement(value=0.0007),
+                    edges=[
+                        geometry["wingtrailingEdge"],
+                        geometry["wingleadingEdge"],
+                        geometry["flaptrailingEdge"],
+                        geometry["flapleadingEdge"],
+                        geometry["slattrailingEdge"],
+                        geometry["slatFrontLEadingEdge"],
+                    ],
+                ),
+                fl.SurfaceEdgeRefinement(
+                    name="symmetry", method=fl.ProjectAnisoSpacing(), edges=[geometry["symmetry"]]
+                ),
+            ],
+        ),
+        reference_geometry=fl.ReferenceGeometry(
+            moment_center=[0.25, 0.005, 0], moment_length=[1, 1, 1], area=0.01
+        ),
+        operating_condition=fl.operating_condition_from_mach_reynolds(
+            mach=0.17, reynolds=1.71e06, temperature=288.16, alpha=8.5 * u.deg, beta=0 * u.deg
+        ),
+        time_stepping=fl.Steady(
+            max_steps=3000, CFL=fl.RampCFL(initial=20, final=300, ramp_steps=500)
+        ),
+        models=[
+            fl.Wall(
+                surfaces=[
+                    geometry["wing"],
+                    geometry["flap"],
+                    geometry["slat"],
+                    geometry["wingTrailing"],
+                    geometry["flapTrailing"],
+                    geometry["slatTrailing"],
+                ],
+                name="wall",
+            ),
```

Comment:
The entities for the fl.Wall() are different from the original case json. Could you also double check to ensure consistent? Same for the other two cases.

Reply:
This is because of the grouping setting that was used, I can change it to a different one where only 3 groupings will exist (wing, flap, slat)